minifi/minifi-nar-bundles/minifi-framework-bundle/pom.xml

```diff
@@ -80,6 +80,21 @@ limitations under the License.
                 <version>2.0.0-SNAPSHOT</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>nifi-property-protection-factory</artifactId>
```

Comment:
See other note on whether this library is required, or whether an alternative simplified version would work better.